Views/Widget-Image.liquid

```diff
@@ -19,7 +22,31 @@
     {% assign cssClasses = cssClasses | append:" " | append: Model.ContentItem.Content.VisibilityPart.ResponsiveVisibility.Text %}
 {% endif %}
 
-<{{tag}} {% if linkTo != blank %}href="{{ linkTo }}"{% endif %} {% if id != blank %}id="{{ id }}"{% endif %} class="{{ cssClasses }} {{ Model.ContentItem | animation_css }}" style="{{ Model.ContentItem | animation_styles }}" {{ Model.ContentItem | animation_data_attributes }}>
+{% if isModal %}
+    {% assign cssClasses = cssClasses | append: ' js-iframe-modal' %}
+{% endif %}
+
+<{{tag}}

```

Comment:
Tidy formatting.

Reply:
do you prefer the formatting be long lines? instead of short lines? just thought it looked more readable

Reply:
I'd like the formatting as I mention in the comment above, it's much easier to digest.

https://github.com/EtchUK/Etch.OrchardCore.Widgets/pull/136/files#r1090380642